python/cuml/internals/input_utils.py

```diff
@@ -44,17 +44,16 @@
     msg='Optional dependency scipy is not installed'
 )
 
-cp_ndarray = gpu_only_import_from('cupy', 'ndarray')
-CudfSeries = gpu_only_import_from('cudf', 'Series')
-CudfDataFrame = gpu_only_import_from('cudf', 'DataFrame')
-DaskCudfSeries = gpu_only_import_from('dask_cudf.core', 'Series')
-DaskCudfDataFrame = gpu_only_import_from('dask_cudf.core', 'DataFrame')
 np_ndarray = cpu_only_import_from('numpy', 'ndarray')
-numba_devicearray = gpu_only_import_from('numba.cuda', 'devicearray')
-try:
-    NumbaDeviceNDArrayBase = numba_devicearray.DeviceNDArrayBase
-except UnavailableError:
-    NumbaDeviceNDArrayBase = numba_devicearray
+PandasSeries = cpu_only_import_from('pandas', 'Series')
+PandasDataFrame = cpu_only_import_from('pandas', 'DataFrame')
+cp_ndarray = gpu_only_import_from('cupy', 'ndarray', alt=np_ndarray)
+CudfSeries = gpu_only_import_from('cudf', 'Series', alt=PandasSeries)
```

Comment:
Why are we using `PandasSeries` as an alt here? (Same question for the rest of the alts)